src/index.ts

```diff
@@ -1,61 +1,37 @@
 import path from 'path';
-import diffDirs, { DirsType, DiffResult, ComparisonOptionsType } from './compare';
-import generateReport from './report';
+import generateReportTemplate from './report';
 import fs from 'fs-extra';
 import mkdirp from 'mkdirp';
+import execa from 'execa';
 
-const showResults = ({
-    failed,
-    passed,
-    diffTime,
-}: {
-    failed: number;
-    passed: number;
-    diffTime: number;
-}) => {
-    const results = `
-    ==================
-    Tests failed: ${failed}
-    Tests passed: ${passed}
-    Diff time: ${diffTime} s.
-    `;
-
-    return results;
-};
-
-type onVrtCompleteType = (result: DiffResult, cmpTime: number) => void;
-
-const onVrtCompleteDefaultAction: onVrtCompleteType = (result, cmpTime) => {
-    const info = showResults({
-        failed: result.failed.length + result.missing.length,
-        passed: result.passed.length + result.new.length,
-        diffTime: cmpTime / 1000,
-    });
-
-    console.info(info);
-};
+// Special log messages for TeamCity service. Using by other platforms
+function teamcityLogger(message: string) {
```

Comment:
can we reuse `teamcityMessage` function from compare - move it out from lexical scope of `diffDirs` to the root level and export it to this file?